typedb/common/exception.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, code: int, message: str):
 MISSING_DB_NAME = ClientErrorMessage(7, "Database name cannot be empty.")
```

Comment:
If `UNKNOWN_REQUEST_ID` had printed out the response, we could've saved some time debugging this time round. We're now printing out the response to aid debugging in the future.

Of course, if it ends up printing sensitive data, the reporter can mask it when reporting the error.